pkg/apis/kops/instancegroup.go

```diff
@@ -116,6 +116,10 @@ type InstanceGroupSpec struct {
 	AdditionalUserData []UserData `json:"additionalUserData,omitempty"`
 	// SuspendProcesses disables the listed Scaling Policies
 	SuspendProcesses []string `json:"suspendProcesses,omitempty"`
+	// LoadBalancers to associate with this instance group (AWS ELB)
```

Comment:
Just noticed this, comment description should match the var name.

`// LoadBalancerNames is a list of load balancers to associate with this instance group (AWS ELB)
LoadBalancerNames []string `json:"loadBalancerNames,omitempty"``